site-map/index.html

```diff
@@ -0,0 +1,156 @@
+                <a href="{{ site.baseurl }}/products/"><h2>Products</h2></a>
+            </div>
+        </div>
+        <div class="row">
+            <div class="small-12 medium-3 columns">
+                <ul class="site-map">
+                    <li><a href="{{ site.baseurl }}/products/">Overview</a></li>
+                </ul>
+            </div>
+            <div class="small-12 medium-9 columns">
+                <a href="{{ site.baseurl }}/omero/"><h3 class="subdivider">&raquo; OMERO</h3></a>
+                <ul class="site-map">
+                    <li><a href="{{ site.baseurl }}/omero/">Overview</a></li>
+                    <li><a href="{{ site.baseurl }}/omero/downloads/">Downloads</a></li>
+                </ul>
+            </div>
+        </div>
+        <div class="row">
+            <div class="small-12 medium-3 medium-offset-3 columns">
+                <a href="{{ site.baseurl }}/omero/scientists/"><h4 class="subdivider">OMERO for Scientists</h4></a>
+                <ul class="site-map">
+                    <li><a href="{{ site.baseurl }}/omero/scientists/">Summary</a></li>
+                    <li><a href="{{ site.baseurl }}/omero/scientists/getting-started.html">Getting Started</a></li>
+                    <li><a href="{{ site.baseurl }}/omero/scientists/docs.html">Documentation</a></li>
+                </ul>
+            </div>
+            <div class="small-12 medium-3 columns">
+                <a href="{{ site.baseurl }}/omero/developers/"><h4 class="subdivider">OMERO for Developers</h4></a>
+                <ul class="site-map">
+                    <li><a href="{{ site.baseurl }}/omero/developers/">Summary</a></li>
+                    <li><a href="{{ site.baseurl }}/omero/developers/getting-started.html">Getting Started</a></li>
+                    <li><a href="{{ site.baseurl }}/omero/developers/docs.html">Documentation</a></li>
+                </ul>
+            </div>
+            <div class="small-12 medium-3 columns">
+                <a href="{{ site.baseurl }}/omero/institution/"><h4 class="subdivider">OMERO for Institutions</h4></a>
+                <ul class="site-map">
+                    <li><a href="{{ site.baseurl }}/omero/institution/">Summary</a></li>
+                    <li><a href="{{ site.baseurl }}/omero/institution/getting-started.html">Getting Started</a></li>
+                    <li><a href="{{ site.baseurl }}/omero/institution/docs.html">Documentation</a></li>
+                </ul>
+            </div>
+        </div>
+        <div class="row">
+            <div class="small-12 medium-9 medium-offset-3 columns">
+                <h4 class="subdivider">Key Features</h4>
+            </div>
+        </div>
+        <div class="row">
+            <div class="small-12 medium-3 medium-offset-3 columns">
+                <ul class="site-map">
+                    <li><a href="{{ site.baseurl }}/omero/new/">NEW (in latest release)</a></li>
```

Comment:
I've tweaked the wording for this section for 5.4.0 so we aren't tied to completely renewing the content every time so this should probably be 'in latest releases' plural

Reply:
@hflynn Sounds good. Thank you!

Reply:
```Lizas-MacBook-Pro:www.openmicroscopy.org liza$ git push lunson site-map
To git@github.com:lunson/www.openmicroscopy.org.git
 ! [rejected]        site-map -> site-map (fetch first)
error: failed to push some refs to 'git@github.com:lunson/www.openmicroscopy.org.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.```

I'm getting rejected. Halp?